Makefile

```diff
@@ -1,5 +1,5 @@
 TEST?=./...
-TESTARGS?=
+TESTARGS?=-race
```

Comment:
Golang has it's own model for flags
https://groups.google.com/d/msg/golang-nuts/3myLL-6mA94/VUkLtSOyS-YJ